Use deploy instead of deploy:deploy

The job "Release Artifacts to Sonatype" failed because of #1433. It throws the following error:

```
Error:  Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:3.1.1:deploy (default-cli) on project kadai-common-logging: The packaging for this project did not assign a file to the build artifact -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
Error:  
Error:  After correcting the problems, you can resume the build with the command
Error:    mvn <args> -rf :kadai-common-logging
Error: Process completed with exit code 1.
```

--- a/.github/workflows/continuous-integration.yml
+++ b/.github/workflows/continuous-integration.yml
@@ -649,8 +649,8 @@ jobs:
           fi
           
           echo "Using Maven profile: $PROFILE"
-          ./mvnw -B deploy:deploy -P $PROFILE \
-          --settings ci/mvnsettings.xml \
+          ./mvnw -B deploy -P $PROFILE \
+          --settings ci/mvnsettings.xml -DskipTests -Dcheckstyle.skip -Djacoco.skip \
           -pl :kadai-parent,\
           :kadai-common-parent,:kadai-common-logging,:kadai-common,:kadai-common-security,\
           :kadai-common-data,:kadai-common-test,:kadai-test-api,\

